### PR TITLE
Email export

### PR DESCRIPTION
**Pull request by ronan-mch**
@abbottjam - Can you have a look at this? I will also need a bit of UI help on the form before putting into prod.
@flyingzumwalt - feel like having a gander?

### CODE REVIEW

File: app/views/catalog/_export_button.html.erb [hunk 1]
```diff
@@ -20,5 +20,9 @@
         <%= link_to content_tag('i', '', :class => 'icon-file-text-o') + t('ddf.tools.ris'), catalog_path(document, :format => 'ris') %>
       </li>
     <% end %>
+
+    <li>
+      <%= link_to t('ddf.tools.email_result'), citations_new_path(id: document.id), data: { ajax_modal: 'trigger'} %>
```
Comment (flyingzumwalt) [inline app/views/catalog/_export_button.html.erb]: shouldn't this be `new_citation_path`, not `citations_new_path`?

Comment (ronan-mch) [inline app/views/catalog/_export_button.html.erb]: probably would be better alright, 'citations_new` was the helper generated by Rails from my route, but I can manually configure something more appropriate.

**Comment by flyingzumwalt** [comment]
Looks good to me.  As a matter of style, you might want to squash your commits.  If you're submitting to hydra repos they always require it, but within DTU it's your call. :wink:

File: app/views/catalog/_export_button.html.erb (hunk 1 above)
Comment (flyingzumwalt) [inline app/views/catalog/_export_button.html.erb]: Go with whatever rails is doing out of the box.  I just thought that it set
up named routes as edit_resource_path, new_resource_path, etc.

Comment (ronan-mch) [inline app/views/catalog/_export_button.html.erb]: Only resource urls i think. As regards commit squashing, I don't see the
benefit of losing that history. What's the argument for doing so?

man. 9. nov. 2015 16.40 skrev flyingzumwalt <notifications@github.com>:

**Pull request closed without merging**